code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -38,7 +38,7 @@
 	new /obj/item/clothing/head/bio_hood/scientist(src)
 	new /obj/item/clothing/under/rank/research_director(src)
 	new /obj/item/clothing/under/dress/dress_rd(src)
-	new /obj/item/clothing/suit/storage/labcoat(src)
+	new /obj/item/clothing/suit/storage/labcoat/rd(src)
```

Comment:
CMOs spawn with a labcoat and have a spare in their locker, is that your plan here too or nah?

Reply:
I intended to just have them suit up from their locker, but it slipped my mind about cmo getting one round start. I will swap their round start worn one with a genetics lab coat.

Reply:
Be sure to add that to the changelog.

Reply:
I will.